source/features/bypass-checks.tsx

```diff
@@ -7,7 +7,12 @@ import * as api from '../github-helpers/api';
 import {getRepoURL} from '../github-helpers';
 
 async function bypass(detailsLink: HTMLAnchorElement): Promise<void> {
-	const runId = new URLSearchParams(detailsLink.search).get('check_run_id') ?? detailsLink.pathname.split('/').pop()!;
+	const runId = pageDetect.isActionJobRun(detailsLink) ? 	detailsLink.pathname.split('/').pop() :// https://github.com/xojs/xo/runs/1104625522
```

Comment:
Rough whitespace around here. There's a tab after `?`

This is the expected indentation for the ternary operator
```suggestion
	const runId = pageDetect.isActionJobRun(detailsLink) ?
		detailsLink.pathname.split('/').pop() : // https://github.com/xojs/xo/runs/1104625522
```